spec/EmailVerificationToken.spec.js

```diff
@@ -288,6 +288,112 @@ describe('Email Verification Token Expiration: ', () => {
       });
   });
 
+  it('can conditionally send emails', async () => {
+    let sendEmailOptions;
+    const emailAdapter = {
+      sendVerificationEmail: options => {
+        sendEmailOptions = options;
+      },
+      sendPasswordResetEmail: () => Promise.resolve(),
+      sendMail: () => {},
+    };
+    await reconfigureServer({
+      appName: 'emailVerifyToken',
+      verifyUserEmails: true,
+      emailAdapter: emailAdapter,
+      emailVerifyTokenValidityDuration: 5, // 5 seconds
+      publicServerURL: 'http://localhost:8378/1',
+    });
+    const beforeSave = {
+      method(req) {
+        expect(req.setSendEmailVerificationEmail).toBeTrue();
+        expect(req.setEmailVerified).toBeFalse();
+        req.setSendEmailVerificationEmail = false;
+        req.setEmailVerified = true;
```

Comment:
Could we just expose the currently internal `Parse.User.emailVerified` field instead of introducing a new  request property `setEmailVerified`?

If we simply exposed it on the [master access level](https://github.com/parse-community/parse-server#access-scopes), maybe Parse Dashboard could also display it as a normal field that can be manipulated with the master key.